openmeetings-core/pom.xml

```diff
@@ -32,6 +32,7 @@
 	<properties>
 		<site.basedir>${project.parent.basedir}</site.basedir>
 		<autoModuleName>apache.openmeetings.core</autoModuleName>
+		<checkstyle.config.location>${project.parent.basedir}/src/config/openmeetings_checkstyle.xml</checkstyle.config.location>
```

Comment:
Needs to overwrite config location inherited from parent. Parent can't configure ${project.parent.basedir} as config as it has no parent. Its the same problem like with site.basedir overwrite 2 lines above. I think this seems to work fine.